src/main/java/seedu/address/model/person/Person.java

```diff
@@ -21,20 +21,39 @@
     private final Name name;
     private final Phone phone;
     private final Email email;
+    private Favourite favourite;
```

Comment:
Possible to set this as `final`?

Reply:
Thanks for the suggestion, made the necessary changes